dataactvalidator/scripts/loadFSRS.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import func
+
+from dataactcore.models.baseInterface import databaseSession
+from dataactcore.models.fsrs import FSRSAward
+from dataactvalidator.fsrs import retrieveAwardsBatch
+
+def loadFSRS(minId):
+    """Fetch and store a batch of Award/Sub-Award records, replacing previous
+    entries."""
+    awards = list(retrieveAwardsBatch(minId))
+    ids = [a.id for a in awards]
+    with databaseSession() as sess:
```

Comment:
Just to make sure I understand this correctly...

The `databaseSession()` you added to `baseInterface.py` provides a way to do a few things in a discrete database session using `WITH...AS`, and this is an alternative to how we've been doing this to date (i.e., creating a new whateverInterface())?